tools/src/zopen_boot_uri.h

```diff
@@ -6,5 +6,11 @@
   #define ZOPEN_BOOT_URI_PREFIX "ZOSOpenTools"
   #define ZOPEN_BOOT_URI_SUFFIX "releases/download/boot"
   #define ZOPEN_BOOT_URI_TYPE "zos.pax.Z"
-  #define ZOPEN_BOOT_PKG { "curl", "gzip", "tar", "make", "git", "meta", NULL }
+
+  /*
+   * Make sure 'meta' is first - we should make all packages independent, but they are not quite yet
+   * We may want to trim down the list of tools a bit. In particular vim and ncurses are not _required_
+   * but they sure are nice.
+   */
+  #define ZOPEN_BOOT_PKG { "meta", "curl", "make", "git", "less", "perl", "jq", "bash", "diffutils", "findutils", "coreutils", "tar", "gzip", "xz", "bzip2", "vim", "ncurses", NULL }
```

Comment:
Should we add coreutils so that we can eliminate asciiecho: https://github.com/ZOSOpenTools/meta/blob/main/bin/lib/zopen-build#L20?

Reply:
Coreutils is there

Reply:
Ah right!